src/app/embed/adapters/merge.adapter.ts

```diff
@@ -0,0 +1,36 @@
+import {Injectable} from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+import { ReplaySubject } from 'rxjs';
+import { QSDAdapter } from './qsd.adapter'
+import { FeedAdapter } from './feed.adapter'
+import { AdapterProperties, hasMinimumParams, DataAdapter, getMergedValues } from './adapter.properties'
+
+const NO_EMIT_YET = Symbol();
+
+@Injectable()
+export class MergeAdapter {
+  private adapters: DataAdapter[]
+  constructor(
+    private FeedAdapter: FeedAdapter,
+    private QSDAdapter:  QSDAdapter
+	) { 
+    this.adapters = [this.QSDAdapter, this.FeedAdapter]
+  }
+
+  getProperties(params): Observable<AdapterProperties> {
+    let chosenAdapters: Observable<AdapterProperties>[];
+
+    chosenAdapters = this.adapters.
+      map(obs => obs.getProperties(params)).
+      map(obs => obs.startWith(NO_EMIT_YET))
+
+    return Observable.combineLatest(...chosenAdapters).map(sources => {
```

Comment:
combineLatest takes a bunch of observables and returns a new observable that emits when all of the observables passed to it have emitted. The reason we call startWith earlier is to ensure that once *any* of the observables have emitted, the combined observable will. Otherwise, it will wait until they all have.